code/__DEFINES/mobs.dm

```diff
@@ -337,7 +337,7 @@
 #define SILENCE_RANGED_MESSAGE (1<<0)
 
 //Respawn timer
-#define RESPAWN_TIMER 6000 // in deciseconds, the time to wait before allowing respawn. 15 minutes = 15 * 60 (seconds in minute) * 15 (seconds to deciseconds)
+#define RESPAWN_TIMER 0 // in deciseconds, the time to wait before allowing respawn. 15 minutes = 15 * 60 (seconds in minute) * 15 (seconds to deciseconds)
```

Comment:
```suggestion
#define RESPAWN_TIMER 600
```
eh make it a minute and see how it goes i guess

Reply:
fair